cyber/python/examples/py_topology.py

```diff
@@ -32,107 +32,107 @@ def __init__(self):
-            print("node {} exist.".format(node_name))
+        if flag is True:
+            print("Node {} exist.".format(node_name))
         else:
-            print("node {} not exist.".format(node_name))
+            print("Node {} does not exist.".format(node_name))
 
     def test_get_node_list(self):
         """
         Test Get Node List.
         """
         node_list = self.test_topology_manager.get_node_list()
         print(len(node_list))
-        if(len(node_list) != 0):
+        if not node_list:
             for node in node_list:
                 print(node)
         else:
-            print("get all nodes failed")
+            print('Failed to get all nodes.')
 
     def test_show_node_info(self, node_name):
         """
         Test Show Node Info.
         """
         ret = self.test_topology_manager.show_node_info(node_name)
-        if(len(ret) != 0):
+        if not ret:
             for item in ret:
-                print item
+                print(item)
 
     def test_get_channel_list(self):
         """
         Test Get Channel List Info.
         """
         channel_list = self.test_topology_manager.get_channel_list()
-        if(len(channel_list) != 0):
+        if not channel_list:
             for channel in channel_list:
                 print(channel)
         else:
-            print("get all channels failed")
+            print('Failed to get all channels.')
 
     def test_get_reader_list(self):
         """
         Test Get Reader List.
         """
         reader_list = self.test_topology_manager.get_reader_list()
-        if(len(reader_list) != 0):
+        if not reader_list:
             for reader in reader_list:
                 print(reader)
         else:
-            print("get all reader failed")
+            print('Failed to get all reader.')
 
     def test_get_writer_list(self):
         """
         Test Get Writer List.
         """
         writer_list = self.test_topology_manager.get_writer_list()
-        if(len(writer_list) != 0):
+        if not writer_list:
             for writer in writer_list:
                 print(writer)
         else:
-            print("get all writer failed")
+            print('Failed to get all writer.')
 
     def test_get_node_writes(self, node_name):
         """
         Test Get Node Writes.
         """
         node_writers = self.test_topology_manager.get_node_writes(node_name)
-        if(len(node_writers) != 0):
+        if not node_writers:
             for node_writer in node_writers:
                 print(node_writer)
         else:
-            print("get all writers of node :{} failed".format(node_name))
+            print('Get all writers of node :{} failed'.format(node_name))
 
     def test_get_node_readers(self, node_name):
         """
         Test Get Node Readers.
         """
         node_readers = self.test_topology_manager.get_node_readers(node_name)
-        if(len(node_readers) != 0):
+        if not node_readers:
```

Comment:
Reversed logic?

Reply:
No. Actually, for sequences(tuple, list, etc...), it's better to check it empty or not in this way in terms of PEP8. 
Please refer to https://www.python.org/dev/peps/pep-0008/

Reply:
I mean, `(len(node_readers) != 0)` indicates non-empty, while `not node_readers` indicates empty.

Reply:
right, will fix it soon. :)